src/csharp/Extensions/Microsoft.Spark.Extensions.FileSystem.E2ETest/FileSystemTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using Microsoft.Spark.E2ETest;
+using Microsoft.Spark.Sql;
+using Microsoft.Spark.UnitTest.TestUtils;
+using Xunit;
+
+namespace Microsoft.Spark.Extensions.FileSystem.E2ETest
+{
+    [Collection(Constants.FileSystemTestContainerName)]
+    public class FileSystemTests
+    {
+        private readonly SparkSession _spark;
+
+        public FileSystemTests(SparkFixture fixture)
+        {
+            _spark = fixture.Spark;
+        }
+
+        /// <summary>
+        /// Test that methods return the expected signature.
+        /// </summary>
+        [Fact]
+        public void TestSignatures()
+        {
+            using FileSystem fs = FileSystem.Get(_spark.SparkContext);
+
+            using var tempDirectory = new TemporaryDirectory();
+            string path = Path.Combine(tempDirectory.Path, "temp-table");
+            _spark.Range(25).Write().Format("parquet").Save(path);
+
+            Assert.True(fs.Delete(path, true));
+            Assert.False(fs.Delete(path, true));
+        }
+    }
```

Comment:
Can we add another test that validates the `Delete` API by checking if the file got deleted from the file system?

Reply:
Changed `TestSignatures()` to only test the signatures, and moved the functional testing into a new test that validates that the file is properly deleted.